Helper/Data.php

```diff
@@ -47,6 +47,7 @@
 use Magento\Store\Model\StoreManagerInterface;
 use Magento\Tax\Model\Calculation;
 use Magento\Tax\Model\Config;
+use function PHPUnit\Framework\isNull;
```

Comment:
why is this necessary?

Reply:
Just a mistake due to the auto completion of Phpstorrm.